warehouse/sponsors/__init__.py

```diff
@@ -22,3 +24,8 @@ def _sponsors(request):
 def includeme(config):
     # Add a request method which will allow to list sponsors
     config.add_request_method(_sponsors, name="sponsors", reify=True)
+
+    # Add a periodic task to update sponsors table
+    config.add_periodic_task(
```

Comment:
We might want to wrap this in a conditional to check if `pythondotorg.api_token` is set, then "unconfigure" it in dev. Mostly since I'm going to suggest we run this hourly to reduce the delay for new sponsors and confusion from PSF staff.